plugins/importexport/native/filter/NativeFilterHelper.inc.php

```diff
@@ -83,32 +122,100 @@ function createCoversNode($filter, $doc, $object) {
 	 * Parse out the object covers.
 	 * @param $filter NativeExportFilter
 	 * @param $node DOMElement
-	 * @param $object Issue|Article
-	 * @param $assocType ASSOC_TYPE_ISSUE | ASSOC_TYPE_SUBMISSION
+	 * @param $object Publication
 	 */
-	function parseCovers($filter, $node, $object, $assocType) {
+	function parsePublicationCovers($filter, $node, $object) {
 		$deployment = $filter->getDeployment();
+
+		$coverImages = array();
+
 		for ($n = $node->firstChild; $n !== null; $n=$n->nextSibling) {
 			if (is_a($n, 'DOMElement')) {
 				switch ($n->tagName) {
 					case 'cover':
-						$this->parseCover($filter, $n, $object, $assocType);
+						$coverImage = $this->parsePublicationCover($filter, $n, $object);
+						$coverImages[key($coverImage)] = reset($coverImage);
 						break;
 					default:
-						$deployment->addWarning($assocType, $object->getId(), __('plugins.importexport.common.error.unknownElement', array('param' => $n->tagName)));
+						$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.unknownElement', array('param' => $n->tagName)));
 				}
 			}
 		}
+
+		$object->setData('coverImage', $coverImages);
+	}
+
+	/**
+	 * Parse out the object covers.
+	 * @param $filter NativeExportFilter
+	 * @param $node DOMElement
+	 * @param $object Issue
+	 */
+	function parseCovers($filter, $node, $object) {
```

Comment:
Maybe this should be `parseIssueCovers`, now that it's specialized to issues? Just for clarity with `parsePublicationCovers`.

Reply:
Fixed.